cmd/protoc-gen-gorums/dev/zorums_types_gorums.pb.go

```diff
@@ -590,3 +593,215 @@ func (c *CorrectableStreamResponse) set(reply *Response, level int, err error, d
 		}
 	}
 }
+
+// ZorumsService is the server-side API for the ZorumsService Service
+type ZorumsService interface {
+	Multicast(*Request)
+	MulticastPerNodeArg(*Request)
+	Multicast2(*Request)
+	Multicast3(*Request)
+	Multicast4(*empty.Empty)
+	OrderingQC(*Request) *Response
+	OrderingPerNodeArg(*Request) *Response
+	OrderingCustomReturnType(*Request) *Response
+	OrderingCombo(*Request) *Response
+	OrderingUnaryRPC(*Request) *Response
+	OrderingFuture(*Request) *Response
+	OrderingFuturePerNodeArg(*Request) *Response
+	OrderingFutureCustomReturnType(*Request) *Response
+	OrderingFutureCombo(*Request) *Response
+	Unicast(*Request)
+	Unicast2(*Request)
+}
+
+func (s *GorumsServer) RegisterZorumsServiceServer(srv ZorumsService) {
+	s.srv.handlers[multicastMethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(Request)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
```

Comment:
Since these unmarshal and marshal options (below) are the same for all, would we save some allocations, if we defined it once, and reused the same in all handlers?